arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -0,0 +1,833 @@
+        },
+        "NamespaceProperties": {
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "The name of the namespace."
+                },
+                "provisioningState": {
+                    "type": "string",
+                    "description": "Provisioning state of the Namespace."
+                },
+                "region": {
+                    "externalDocs": {
+                        "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+                    },
+                    "type": "string",
+                    "description": "Specifies the targeted region in which the namespace should be created. It can be any of the following values: Australia EastAustralia SoutheastCentral USEast USEast US 2West USNorth Central USSouth Central USEast AsiaSoutheast AsiaBrazil SouthJapan EastJapan WestNorth EuropeWest Europe"
+                },
+                "status": {
+                    "externalDocs": {
+                        "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+                    },
+                    "type": "string",
+                    "description": "Status of the namespace. It can be any of these values:1 = Created/Active2 = Creating3 = Suspended4 = Deleting"
+                },
+                "createdAt": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The time the namespace was created."
+                },
+                "serviceBusEndpoint": {
+                    "externalDocs": {
+                        "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+                    },
+                    "type": "string",
+                    "description": "Endpoint you can use to perform ServiceBus operations."
+                },
+                "subscriptionId": {
+                    "type": "string",
+                    "description": "The Id of the Azure subscription associated with the namespace."
+                },
+                "critical": {
+                    "type": "boolean",
+                    "description": "Whether or not the namespace is set as Critical."
+                },
+                "namespaceType": {
+                    "type": "string",
+                    "description": "The namespace type.",
+                    "enum": [
+                        "Messaging",
+                        "NotificationHub",
+                        "EventHub"
+                    ],
+                    "x-ms-enum": {
+                        "name": "NamespaceType",
+                        "modelAsString": false
+                    }
+                }
+            },
+            "description": "Namespace properties."
+        },
+        "RegenerateKeysParameters": {
+            "properties": {
+                "Policykey": {
```

Comment:
What value should the user provide? Primary or Secondary or something else? If you can update the description to clarify this then that would be great. If it is indedd Primary or Secodary or a set of predefined values then should this be an enum to make it easy for the customer?